### PR TITLE
Add rocks for designate

### PR DESCRIPTION
**Pull request by gboutry**
ROCKs added:
  - designate-api
  - designate-central
  - designate-consolidated
  - designate-mdns
  - designate-producer
  - designate-worker

### CODE REVIEW

**Comment by gboutry** [comment]
Draft until following PRs are merged:
- https://github.com/canonical/ubuntu-openstack-rocks/pull/2

File: rocks/designate-consolidated/rockcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        --shell /bin/false \
+        designate
+
+  designate-consolidated:
+    after: [designate-user]
+    plugin: nil
+    overlay-packages:
+      - sudo
+      - apache2
+      - libapache2-mod-wsgi-py3
```
Comment (hemanthnakkina) [inline rocks/designate-consolidated/rockcraft.yaml:39]: This package is not listed in designate-api.

**Review by hemanthnakkina — CHANGES_REQUESTED** [review]

File: rocks/designate-consolidated/rockcraft.yaml (hunk 1 above)
Comment (gboutry) [reply rocks/designate-consolidated/rockcraft.yaml:39]: Fixed

**Review by hemanthnakkina — APPROVED** [review]
LGTM

**Pull request merged**